rewriter/Concern.cc

```diff
@@ -0,0 +1,152 @@
+#include "rewriter/Concern.h"
+#include "ast/Helpers.h"
+#include "ast/ast.h"
+#include "core/Names.h"
+#include "core/core.h"
+
+using namespace std;
+
+namespace sorbet::rewriter {
+
```

Comment:
```suggestion

namespace {

```

Reply:
These functions all have public (non-static) linkage, which can cause confusing linker errors if someone else happens to write a function with the same name.